tests/Lykke.Service.FixGateway.Tests/TradeSessionIntegration/TradeSessionExternalIntegrationTest.cs

```diff
@@ -87,6 +103,58 @@ public static void ShouldPlaceMarketOrder(FixClient fixClient, NewOrderSingle or
             Assert.That(ex.LastPx.Obj, Is.GreaterThan(0));
         }
 
+        public static void ShouldPlaceLimitOrder(FixClient fixClient, NewOrderSingle orderRequest)
+        {
+            fixClient.Send(orderRequest);
+
+            var response = fixClient.GetResponse<Message>();
+
+            Assert.That(response, Is.Not.Null);
+            Assert.That(response, Is.TypeOf<ExecutionReport>());
+
+            var ex = (ExecutionReport)response;
+            Assert.That(ex.OrdStatus.Obj, Is.EqualTo(OrdStatus.PENDING_NEW));
+            Assert.That(ex.ExecType.Obj, Is.EqualTo(ExecType.PENDING_NEW));
+
+
+            Thread.Sleep(1000000);
```

Comment:
It is inappropriate for unit test to wait 1000 sec.

Reply:
Mark the test as Explicit. It will not run on CI